pkg/crc/preflight/preflight_darwin.go

```diff
@@ -16,6 +21,12 @@ func StartPreflightChecks() {
 
 // SetupHost performs the prerequisite checks and setups the host to run the cluster
 func SetupHost() {
+	preflightCheckAndFix(false,
```

Comment:
Should we also have some `skip-check-oc-binary-cached` config property?

Reply:
@jsliacan no as I said this is required for post vm part so we are always going to need it.

Reply:
we can not skip this...